src/olympia/abuse/tests/test_utils.py

```diff
@@ -332,6 +334,7 @@ def setUp(self):
         self.addon = addon_factory(users=(self.author,), name='<b>Bad Addön</b>')
         ActivityLog.objects.all().delete()
         self.cinder_job.abusereport_set.update(guid=self.addon.guid)
+        self.create_switch('enable-cinder-reviewer-tools-integration', active=True)
```

Comment:
could decorate the class with  `@override_switch('enable-cinder-reviewer-tools-integration', active=True)` instead